### DISCUSSION TITLE
StorageManager changes & dynamic endpoints

### DISCUSSION THREAD

**Discussion by Bran72** [GrapesJS/grapesjs#4442] [Q&A]
Hey @artf !

Since the v0.19.4, I'm unable to dynamically change the endpoints of my remote storage.
Previously, I did it as follow:
`this.editor.StorageManager.get('remote').set({
   urlStore: urlStore,
   urlLoad: urlLoad
})`

But this code won't work anymore, so Could you tell me how I can reproduce the same ?
I tried the following, but it's not working as expected. I think the endpoints are well replaced 'cause when I make an update, it POST on the right route. But when I try to load the content when my endpoints change, `loadedProjectData` allways gives me an empty object...
```
this.editor.Storage.add('remote', {
   async load () {
      return await axiosLaravel.get(urlLoad)
   },
   async store (data) {
      return await axiosLaravel.patch(urlStore, data)
   }
})
const loadedProjectData = await this.editor.load()
this.editor.loadProjectData(loadedProjectData)
```

Thx for your time !

**Comment by artf** [comment]
I think it depends on how you update the `urlLoad`, are you sure when the load starts the `urlLoad` is actually updated?

ps. ` this.editor.load()` updates already the editor with new data, you don't need to update it again with `this.editor.loadProjectData`
